src/client/src/commons/map/overlay/mapOverlayCmp.js

```diff
@@ -108,62 +108,6 @@ class MapOverlayComponent extends React.Component {
                   />
                 </div>
                 <div>
-                  {setSelectedLayer === undefined ? null : layer.queryable ===
```

Comment:
REQ: Das Property `queryable` wurde vermutlich nur in diesem Kontext verwendet und sollte daher gelöscht werden. Bitte prüfen, ob es noch andere Properties gibt, die nur für diesen Info-Button verwendet wurden und gelöscht werden können (z.B. `Identifier`).

Reply:
Ich habe den auskommentieren Code gelöscht. Ausserdem habe ich noch weitere props gelöscht.